src/Card/CardHeader.jsx

```diff
@@ -6,6 +6,7 @@ import CardActions from './CardActions';
 import buildThemeProperties from '../Utils';
 
 import style from '../Layout/LayoutStyle.scss';
+import PlaceholderTextComponent from '../PlaceholderComponent';
```

Comment:
no need to import this... the PlaceholderComponent element will be passed in as a prop (it's up to the parent/containing component to create the PlaceholderComponent and pass it in as the title prop